plutus-book/doc/09-extended.adoc

```diff
@@ -325,3 +325,17 @@ rolled back transaction. We do not give the details of this functionality here.
 
 There are other features of the extended UTxO system that are less relevant to
 a Plutus user, which we will also not explain in detail in this chapter.
+
+.Adding Ledger Functionality Using Scripts
```

Comment:
This should be a level-3 header - I changed the other ones in this section but you'd deleted this one ;)